ddtrace/contrib/grpc/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .patch import patch, unpatch
+__all__ = ['patch', 'unpatch']
```

Comment:
You will want to follow a similar pattern as [this](https://github.com/DataDog/dd-trace-py/blob/612189378169eea35b94f52af7d9f1f2b0dd0622/ddtrace/contrib/vertica/__init__.py)

We want to have some checks to ensure `grpc` is installed/available.